console/working-on-tickets.rst

```diff
@@ -145,3 +145,32 @@ Before being able to use the new states within the WebApp, you need to run the f
     attribute.screens[:edit]['ticket.customer'][:filter] = Ticket::State.by_category(:viewable_customer_edit).pluck(:id)
     attribute.save!
 
+
+Limit available states for customers
+------------------------------------
+
+By default Zammad allows customers to change Ticket states to ``open`` and ``closed``. 
+If this does not meet your requirenments, you can adjust this at anytime.
+The below example shows how to restrict your customer to only close tickets if needed:
+
+::
+  
+  attribute = ObjectManager::Attribute.find_by(name: 'state_id')
```

Comment:
I had a look at the overall document and found out that we usually prefer the `.get` method for picking the right attribute. We should this apply here as well:

```diff
- attribute = ObjectManager::Attribute.find_by(name: 'state_id')
+ attribute = ObjectManager::Attribute.get(
+   object: 'Ticket',
+   name: 'state_id',
+ )
```

Reply:
fixed on https://github.com/zammad/zammad-documentation/pull/105/commits/73dd59f986191204469ffe19bf1cbaee0cac3ecb